.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,5 +1,8 @@
-#Tue Jan 24 17:02:29 PST 2012
 eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.6
```

Comment:
If we cannot rely on Eclipse understanding the `pom` correctly, then we should version Eclipse config files ...